concent-builder/repository-prepare.yml

```diff
@@ -0,0 +1,59 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+  tasks:
+    - become:      yes
+      remote_user: builder
+      block:
+        - name:    Create a directory for secrets
+          file:
+            path:  "/home/{{ shared_user }}/build/concent-secrets"
+            state: directory
+
+        - name:    Copy secrets to remote location
+          copy:
+            src:   "{{ local_secret_dir }}/{{ item }}"
+            dest:  "/home/{{ shared_user }}/build/concent-secrets"
```

Comment:
You'll need to run this particular task as `root` and add `owner` and `mode` parameters to give the files to `shared_user`.

I've investigated the issues we had with running `copy` as non-root and the cause seems to be a bug in Ansible (see their issue 21145). We have to work around it.